demisto_sdk/commands/common/hook_validations/modeling_rule.py

```diff
@@ -209,3 +215,30 @@ def is_valid_rule_names(self):
                 return False
 
         return True
+
+    def is_valid_rule_suffix_name(self):
+        """
+        Verifies the following::
+            1. The modeling rule file name ends with 'ModelingRule.yml'.
```

Comment:
```suggestion
            1. The modeling rule file name ends with MODELING_RULE_FILE_SUFFIX.
```
In my opinion, it is best to use constants in the comment, so that if they change, the comment won't need to be modified